engine/swarm/how-swarm-mode-works/services.md

```diff
@@ -76,6 +76,27 @@ schedules tasks to worker nodes.
 
 ![services flow](../images/service-lifecycle.png)
 
+### Pending services
+
+You can even configure a service in such a way that no node currently in the
+swarm can run its tasks. Here are a few examples:
+
+- You can even create a service with an image that is not available in the
+  registry, either because it hasn't been pushed to the registry yet or because
+  the registry is not currently available.
```

Comment:
Using a non-existing image is actually dangerous, as (contrary to some other constraints), swarm does not know about this when scheduling, so it will keep trying to deploy the task, and pull images. It's possible the node runs out of resources because of this, or (if no log-rotation is in place) runs out of disk space.